python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -1931,10 +2002,19 @@ def _remeasure_star_background(self, result, background_to_photometric_ratio=Non
             background=result.background,
             backgroundToPhotometricRatio=background_to_photometric_ratio,
         ).background
-        # Isolate the final pedestal background to log the computed value
+        # Isolate the final pedestal background to log the computed value. Set
+        # a maximum allowable to avoid unnaturally high values (this step is
+        # largely trying to correct for oversubtraction).
         pedestalBackground = afwMath.BackgroundList()
         pedestalBackground.append(pedestalBackgroundList[1])
         pedestalBackgroundLevel = pedestalBackground.getImage().array[0, 0]
+        if pedestalBackgroundLevel > self.config.star_background_max_pedestal:
+            self.log.warning("Computed final background pedestal (%.4f) was larger than the "
+                             "maximum allowed (%.2f).  Setting it to %.2f",
+                             pedestalBackgroundLevel, self.config.star_background_max_pedestal,
+                             self.config.star_background_max_pedestal)
+            pedestalBackgroundLevel = self.config.star_background_max_pedestal
+
         self.log.warning("Subtracted pedestalBackgroundLevel = %.4f", pedestalBackgroundLevel)
```

Comment:
Please change this to an info level log.  If this is normal operations, it's not a warning.